sources/functions/qbittorrent

```diff
@@ -15,21 +14,17 @@ function whiptail_qbittorrent() {
         case ${codename} in
             "xenial" | "stretch")
                 function=$(whiptail --title "Install Software" --menu "Choose a qBittorrent version:" --ok-button "Continue" --nocancel 12 50 3 \
-                    "Repo" "(${repov})" \
                     "4.1" "(${latestv41})" 3>&1 1>&2 2>&3)
                 ;;
             *)
                 function=$(whiptail --title "Install Software" --menu "Choose a qBittorrent version:" --ok-button "Continue" --nocancel 12 50 4 \
-                    "Repo" "(${repov})" \
                     "4.1" "(${latestv41})" \
                     "4.2" "(${latestv42})" \
                     "4.3" "(${latestv43})" 3>&1 1>&2 2>&3)
                 #"Latest" "(${latestv})" 3>&1 1>&2 2>&3)
```

Comment:
No, the option is now baked in above. If you change the method between repo and compile you will need to `export LIBTORRENT_RASTERBAR_METHOD=repo` before running the upgrade script (this will need to be clarified in the docs).  I don't see people switching between repo and compile often once chosen so I don't want this popup running every time.